### DISCUSSION TITLE
blink code stops accepting key input

### DISCUSSION THREAD

**Discussion by crazyelectron-io** [blinksh/blink#1838] [Support]
While using code in blink it frequently stops accepting keyboard input on my iPad. Sometimes it gets restored when reopening the editor session again but most of the time I have to stop blink altogether to get working again for a while.
I am using the blink+code version.

**Comment by yury** [comment]
Hello @crazyelectron-io 

I created issue for this #1839.

Few questions:

1. Are you editing files with blink-fs or code-server? 
2. If you use code-server - what code-server?
3. Do you use software or hardware kb?

**Comment by crazyelectron-io** [comment]
Hi,
I’m using blink-fs and Apple Magic Keyboard.

Thanks for creating an issue.
